src/platform/forms-system/src/js/review/SubmitController.jsx

```diff
@@ -44,13 +44,7 @@ class SubmitController extends Component {
   };
 
   handleSubmit = () => {
-    const {
-      form,
-      formConfig,
-      pageList,
-      trackingPrefix,
-      inProgressFormId,
-    } = this.props;
+    const { form, formConfig, pageList, trackingPrefix } = this.props;
```

Comment:
`inProgressFormId` is found in `form`, so I removed it from `mapStateToProps` in favor of grabbing it only when we needed it. No functionality was changed here—in fact, I wrote a test to ensure that. :sweat_smile: